src/views/Stack/Card.tsx

```diff
@@ -129,6 +133,9 @@ export default class Card extends React.Component<Props> {
   private isVisible = new Value<Binary>(TRUE);
   private isVisibleValue: Binary = TRUE;
   private nextIsVisible = new Value<Binary | -1>(UNSET);
+  private isVerticalGestureDirectionInverted = new Value<Binary>(
+    this.props.gestureDirection === 'vertical-inverted' ? TRUE : FALSE
```

Comment:
where do you update this value? Could you follow https://github.com/react-native-community/react-native-tab-view/blob/6198d423e6c7d618b9abd371a91615b18cc7ca4c/src/Pager.tsx#L138?

Reply:
and verify it somehow?